pkg/resources/files/etcfile_spec.go

```diff
@@ -0,0 +1,80 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package files
+
+import (
+	"fmt"
+	"io/fs"
+
+	"github.com/cosi-project/runtime/pkg/resource"
+	"github.com/cosi-project/runtime/pkg/resource/meta"
+)
+
+// EtcFileSpecType is type of EtcFile resource.
+const EtcFileSpecType = resource.Type("EtcFileSpecs.net.talos.dev")
```

Comment:
Here we have `net`, but the namespace the resource resides is `files`. Not saying it's wrong, just asking if that aligns with our standard for the naming.

Reply:
yep, thanks, should be `files` to keep things consistent